isis/src/qisis/objs/ImageFileListWidget/ImageFileListWidget.cpp

```diff
@@ -621,10 +623,11 @@ namespace Isis {
    */
   void ImageFileListWidget::save(QXmlStreamWriter &stream, Project *project,
                                  FileName newProjectRoot) const {
+
     stream.writeStartElement("imageFileList");
 
     // Write QSettings
-    stream.writeStartElement("widgetGeometry");
+//  stream.writeStartElement("widgetGeometry");
```

Comment:
Why is this uncommented but the rest of this block is not? Can the rest of this be deleted?